app/views/api/v1/shared/_user_show_extended.json.jbuilder

```diff
@@ -17,3 +17,4 @@ end
 
 json.joined_at     I18n.l(user.created_at, format: :json)
 json.profile_image user.profile_image_url_for(length: 320)
+json.badge_ids     user.badge_ids
```

Comment:
Wanted to use postgres `array_agg` , but decided to keep the code simple and just have 1 more sql query (as it's for one user).